### DISCUSSION TITLE
Malfunctional Institutional Email account.

### DISCUSSION THREAD

**Discussion by sergyDwhiz** [community/community#60828] [GitHub Education]
### Select Topic Area

Question

### Body

Dear Sir/Madam,

I am writing to bring to your attention that the institutional email accounts provided to us as students at the University of Buea ([ubuea.cm](http://ubuea.cm/)) are not functioning properly. As a result, students at the University of Buea are unable to take advantage of great opportunities such as the Developer Students Pack and Campus Ambassador roles. I kindly request that you provide us with alternative methods to apply for these opportunities.

Thank you for your prompt attention to this matter.

**Comment by tahgoody** [comment]
This is looks simple yet it importance and relevance should not be neglected. Faced this 2 years ago still surprised it hasn’t yet been fixed

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Sergius-Nyah, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
